openedx/core/djangoapps/credit/models.py

```diff
@@ -467,6 +467,28 @@ def add_or_update_requirement_status(cls, username, requirement, status="satisfi
             requirement_status.reason = reason if reason else {}
             requirement_status.save()
 
+    @classmethod
+    @transaction.commit_on_success
+    def remove_requirement_status(cls, username, requirement, status="satisfied"):
+        """
+        Remove credit requirement status for given username.
+
+        Args:
+            username(str): Username of the user
+            requirement(CreditRequirement): 'CreditRequirement' object
+            status(str): Status of the requirement
+
+        """
+
+        requirement_status = cls.objects.filter(
```

Comment:
We do not need the status here. I am going to remove the status parameter from the method.